tests/artifacts/scoring-blueprints-submission/.mcpd.toml

```diff
@@ -1,8 +1,8 @@
 [[servers]]
   name = "slack"
   package = "npx::@modelcontextprotocol/server-slack@latest"
-  tools = ["slack_post_message", "slack_list_channels"]
-  required_env = ["SLACK_BOT_TOKEN", "SLACK_TEAM_ID"]
+  tools = ["slack_list_channels", "slack_post_message"]
```

Comment:
I don't get if there is a reason for reversing this list or it is just an aesthetic change.